Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -134,23 +134,37 @@ public class NewGcsBucketDynamicParameters
 
             /// <summary>
             /// <para type="description">
-            /// Default ACL for the bucket. e.g. "publicRead", "private", etc.
+            /// Default ACL for the bucket.
```

Comment:
Not required but I find both of these are a little hard to parse. It would be nice if enum values and "*** permission" were quoted.